.github/ISSUE_TEMPLATE/Release_FOSS_ComplianceTemplate.md

```diff
@@ -1,5 +1,5 @@
 ---
-name: FOSS Release Compliance 23.12
+name: FOSS Release Compliance 24.03
```

Comment:
Hi @kelaja 

thanks for updating the Issue Templates, i think we should remove the CalVer from the issue templates

Reply:
Hi @FaGru3n, yes this will be removed for 24.05. Release, we are currently in the process to define this general valid Templates for 24.05. For 24.03 we promised the FOSS-teams no additional process change, therefore please approve.